shared/tracker/header.render.desktop.js

```diff
@@ -16,51 +17,18 @@ export default class HeaderRender extends Component {
   }
 
   render () {
-    let headerStyle = this.props.currentlyFollowing ? styleHeaderFollowing : styleHeaderNotFollowing
-    let headerTextStyle = styleHeaderTextNormal
+    const isWarningAboutTrackerShowingUpLater = this.props.loggedin && !this.props.currentlyfollowing && this.state.showCloseWarning
+    const headerText = isWarningAboutTrackerShowingUpLater ? 'You will see this window every time you access this folder.' : this.props.reason
 
-    if (this.props.currentlyFollowing) {
-      switch (this.props.trackerState) {
-        case 'warning':
-          headerStyle = styleHeaderWarning
-          headerTextStyle = styleHeaderTextWarning
-          break
-        case 'error': headerStyle = styleHeaderError; break
-      }
-    }
-
-    let headerText: string = this.props.reason
-    let isWarning = false
-    if (this.props.loggedIn && !this.props.currentlyFollowing && this.state.showCloseWarning) {
-      isWarning = true
-      headerStyle = styleHeaderWarning
-      headerTextStyle = styleHeaderTextWarning
-      headerText = 'You will see this window every time you access this folder.'
-    }
-
-    // If there's a lastAction, it overrides everything else.
-    switch (this.props.lastAction) {
```

Comment:
lastAction display logic was removed because it was leading to more problems than it was fixing. It may have allowed the header to display the right color, but it didn't fix the crux of the issue which was the proofs being out of date. We have a ticket open for making sure that the proos stay up to date here [DESKTOP-1582](https://keybase.atlassian.net/browse/DESKTOP-1582)